src/phpDocumentor/Descriptor/Builder/Reflector/Tags/SeeAssembler.php

```diff
@@ -65,4 +64,37 @@ public function create($data)
 
         return $descriptor;
     }
+
+    /**
+     * @param $context
+     * @param $referenceParts
+     * @return mixed
+     */
+    private function setFirstReferencePartAsType($context, $referenceParts)
+    {
+        $type = current($referenceParts);
+        $type = new Collection(
+            array($type),
+            $context
+        );
+        $referenceParts[0] = $type;
+        return $referenceParts;
+    }
+
+    /**
+     * When you have a relative reference to a class, we need to check if this class exists in the namespace aliases
+     *
+     * @param $reference
+     * @param $context
+     * @return bool
+     */
+    private function referenceIsNamespaceAlias($reference, $context)
+    {
+        /** @var \phpDocumentor\Reflection\DocBlock\Context $context*/
+        foreach ($context->getNamespaceAliases() as $alias) {
+            if (substr($alias, -strlen($reference), strlen($reference)) === $reference) {
```

Comment:
substr with a negative value will already return the string from the end. So I think you can remove the third param here.